tests/framework/e2e.go

```diff
@@ -17,13 +17,17 @@ package framework
 import (
 	"context"
 	"os"
+	"strings"
 	"testing"
+	"time"
 
 	"go.etcd.io/etcd/client/pkg/v3/testutil"
 	"go.etcd.io/etcd/tests/v3/framework/config"
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
+const TickDuration = 10 * time.Millisecond
```

Comment:
minor comment: the `TickDuration` is only used in `e2e.go`, no need to expose it (the first letter uppercase).

I am thinking probably it makes more sense to just define one public `const` to be used for both e2e and integration ( [cluster.go#L72](https://github.com/etcd-io/etcd/blob/fff5d00ccfddaf5f06fa11cad65c1ecaa8c09f87/tests/framework/integration/cluster.go#L72) ) . Of course, we can address this in a separate PR.

Reply:
It sound good👍
#14344 was opened to address this.